content/source/docs/cloud/api/changelog.html.md

```diff
@@ -14,6 +14,11 @@ page_id: "api-changelog"
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
 
+### 2021-08-18
+
+* Introduced the [State Version Outputs](./state-versions.html) endpoint to retrieve the Outputs for a
```

Comment:
I'm definitely late to this, but this is a tad confusing, because we have a separate [State Version Outputs API](https://www.terraform.io/docs/cloud/api/state-version-outputs.html).

Thoughts on changing this to the following?

```markdown
* [State Versions](https://www.terraform.io/docs/cloud/api/state-versions.html): added [List State Version Outputs](https://www.terraform.io/docs/cloud/api/state-versions.html#list-state-version-outputs) endpoint.
```

This is also inline with the rest of the API changelog formatting.